templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -124,6 +132,11 @@ contents:
         extra_phys_args="dev ${vlan_parent} id ${vlan_id}"
       elif [ $(nmcli --get-values connection.type conn show ${old_conn}) == "bond" ]; then
         iface_type=bond
+        # check bond options
+        bond_opts=$(nmcli --get-values bond.options conn show ${old_conn})
+        if [ -n "$bond_opts" ]; then
+          extra_phys_args+="bond.options ${bond_opts} "
```

Comment:
This is going to have quoting issues. The `bond.options` value can definitely have spaces in it, and maybe other shell metacharacters.

Reply:
I don't think bond.options can have spaces:
```
[trozet@trozet ~]$ nmcli conn add type bond con-name bond1 bond.options "mode=active-backup miimon=100 fail_over_mac=follow"
Error: failed to modify bond.options: 'active-backup miimon=100 fail_over_mac=follow' not among [balance-rr, active-backup, balance-xor, broadcast, 802.3ad, balance-tlb, balance-alb].
[trozet@trozet ~]$ 
[trozet@trozet ~]$ nmcli conn add type bond con-name bond1 bond.options mode=active-backup miimon=100 fail_over_mac=follow
Error: invalid <setting>.<property> 'miimon=100'.
[trozet@trozet ~]$ nmcli conn add type bond con-name bond3 bond.options mode=active-backup,miimon=100,fail_over_mac=follow
Connection 'bond3' (4fefd985-fbb9-4d16-a51d-fd28361e5cae) successfully added.
```

ifcfg bond options can have spaces:
BONDING_OPTS="mode=active-backup miimon=100 fail_over_mac=follow"

however when the ifcfg is loaded into NM, NM is smart enough to convert it to csv

Reply:
after some experimentation it looks like this should be ok